test/spec/extension.js

```diff
@@ -0,0 +1,253 @@
+
+    it('should allow on ServiceTask', function() {
+
+      // given
+      var ioMapping = moddle.create('zeebe:IoMapping'),
+          input = moddle.create('zeebe:Input'),
+          serviceTask = moddle.create('bpmn:ServiceTask'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      ioMapping.$parent = extensionElements;
+      extensionElements.$parent = serviceTask;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(input, extensionElements);
+
+      // then
+      expect(canCopyProperty).not.to.be.false;
+    });
+
+
+    it('should not allow on ReceiveTask', function() {
+
+      // given
+      var ioMapping = moddle.create('zeebe:IoMapping'),
+          input = moddle.create('zeebe:Input'),
+          receiveTask = moddle.create('bpmn:ReceiveTask'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      ioMapping.$parent = extensionElements;
+      extensionElements.$parent = receiveTask;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(input, extensionElements);
+
+      // then
+      expect(canCopyProperty).to.be.false;
+    });
+
+
+    it('should not allow on Task', function() {
+
+      // given
+      var ioMapping = moddle.create('zeebe:IoMapping'),
+          input = moddle.create('zeebe:Input'),
+          task = moddle.create('bpmn:Task'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      ioMapping.$parent = extensionElements;
+      extensionElements.$parent = task;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(input, extensionElements);
+
+      // then
+      expect(canCopyProperty).to.be.false;
+    });
+  });
+
+
+  describe('<zeebe:Output>', function() {
+
+    it('should allow on ServiceTask', function() {
+
+      // given
+      var ioMapping = moddle.create('zeebe:IoMapping'),
+          output = moddle.create('zeebe:Output'),
+          serviceTask = moddle.create('bpmn:ServiceTask'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      ioMapping.$parent = extensionElements;
+      extensionElements.$parent = serviceTask;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(output, extensionElements);
+
+      // then
+      expect(canCopyProperty).not.to.be.false;
+    });
+
+
+    it('should allow on ReceiveTask', function() {
+
+      // given
+      var ioMapping = moddle.create('zeebe:IoMapping'),
+          output = moddle.create('zeebe:Output'),
+          receiveTask = moddle.create('bpmn:ReceiveTask'),
+          extensionElements = moddle.create('bpmn:ExtensionElements');
+
+      ioMapping.$parent = extensionElements;
+      extensionElements.$parent = receiveTask;
+
+      // when
+      var canCopyProperty = zeebeModdleExtension.canCopyProperty(output, extensionElements);
+
+      // then
+      expect(canCopyProperty).not.to.be.false;
```

Comment:
`to.be.true` is the appropriate shortcut here.

Reply:
to.be.undefined is an option, too.

Reply:
Pattern from [`camunda-bpmn-moddle`](https://github.com/camunda/camunda-bpmn-moddle/blob/master/test/spec/extension.js#L41). 

Both `true` and `undefined` values are correct from the point of the outcome so I would not require to be super-specific here.